adam/learner/surface_templates.py

```diff
@@ -120,14 +154,12 @@ def match_against_tokens(
                             slot_filler_span.start : slot_filler_span.end
                         ]
                     )
-                # if there's a slot without a filler, we just don't learn from this instance instead of raising a runtime error
-                else:
-                    return None
-                    # raise RuntimeError(
-                    #    f"Template contained variable {element}, "
-                    #    f"but it was not found in the mapping of slots to spans: "
-                    #    f"{slots_to_filler_spans}"
-                    # )
+                # else:
```

Comment:
Why is this commented out? Do we still need it? Is it commented out until some issue can be fixed? Is it no longer relevant?